### PR TITLE
Lab option default setup and expired link fix

### PR DESCRIPTION
**Pull request by Shreets**
-> set  `Take a Picture` as default lab results input page
-> show thank you message for expired link
-> show input type date's default date editor only when value is selected in browser

### CODE REVIEW

File: src/pages/Submission.js [hunk 1]
```diff
@@ -2,35 +2,56 @@ import React, { useEffect } from "react";
...
           </div>
-          <br />
-          <br />
-          <div>
-            You may close the window.
-          </div>
-        </div>
-      </div>
+        </>
+      )}
```
Comment (nastikue) [inline src/pages/Submission.js:54]: please make condition for <div className="text-greeting"> content only as all the rest is similar

File: src/pages/home.css [hunk 2]
```diff
@@ -273,16 +273,23 @@ input[type="date"]:disabled {
   background-color: #e0e3ea;
 }
 
-input[type="date"]:not(.has-value):before{
+input[type="date"]:not(.has-value):before {
   opacity: 0.3;
   content: attr(placeholder);
 }
 
+input[type="date"]:not(.has-value)::-webkit-datetime-edit-text,
```
Comment (nastikue) [inline src/pages/home.css:281]: 👍

File: src/pages/Submission.js (hunk 1 above)
Comment (Shreets) [reply src/pages/Submission.js]: resolved

File: src/pages/Submission.js [hunk 3]
```diff
@@ -2,35 +2,49 @@ import React, { useEffect } from "react";
...
+                ? "You have already submitted the form."
+                : "You will hear from us shortly."}
+            </div>
+            <br />
+            <br />
+            <div>
+              {messageStatus === MESSAGE_STATUS.processed
+                ? ""
+                : "You may close the window."}
+            </div>
```
Comment (nastikue) [inline src/pages/Submission.js:44]:
```suggestion
            {messageStatus === MESSAGE_STATUS.processed
              ? (<div>
                <p className="submission-title">
                  <b className="bold-text">Thank you</b>
                </p>
                <p className="submission-lead-paragraph">You have already submitted the form.</p>
              </div>)
              : (<div>
                <p className="submission-title">
                  <b className="bold-text">Thank you</b>{" for updating your health information."}</p>
                <p className="submission-lead-paragraph">You will hear from us shortly.</p>
                <p>You may close the window.</p>
              </div>)
            }
```

Comment (Shreets) [reply src/pages/Submission.js]: suggested changes have been  made

**Pull request merged**